concent_api/core/payments/backends/sci_backend.py

```diff
@@ -81,6 +82,13 @@ def get_list_of_payments(
             from_block=first_block_after_payment_number,
             to_block=latest_block_number - payment_interface.REQUIRED_CONFS,  # pylint: disable=no-member
         )
+    elif transaction_type == TransactionType.FORCE:
+        payments_list = payment_interface.get_forced_subtask_payments(  # pylint: disable=no-member
+            payer_address=Web3.toChecksumAddress(requestor_eth_address),
+            payee_address=Web3.toChecksumAddress(provider_eth_address),
+            from_block=first_block_after_payment_number,
+            to_block=latest_block_number - payment_interface.REQUIRED_CONFS,  # pylint: disable=no-member
```

Comment:
@cameel Should it be also added in `get_covered_additional_verification_costs`? I am asking because it was not mentioned in review of https://github.com/golemfactory/concent/pull/1087 and it was already merged, so if it is needed (and it is according to linked comment) I can still add it here.

Reply:
Yeah, good point. It should.

I did not notice because in the blueprint I did not have to pay attention to the verification payments. The the same rule applies here - they don't have `closure_time` so we're relying on blockchain timestamps. It we want to be sure that we get all the payments after a specific point in time we have to use a safety margin of a few blocks before that point.

Anyway, since the timestamp used as the beginning of the the report is only a heuristic. it won't change much in practice but I'd would be better to have it for consistency and in case someone does have to use it for something else in the future. It's not obvious that such a margin is necessary.

Please also add a comment explaining why it's done like this.

Reply:
:+1: done